src/Core/Domain/Carrier/Command/AddCarrierCommand.php

```diff
@@ -41,7 +41,12 @@ public function __construct(
         private string $trackingUrl,
         private int $position,
         private bool $active,
+        private array $associatedGroupIds,
         private ?string $logoPathName,
```

Comment:
```suggestion
        private ?string $logoPathName = null,
```
Optional, can be done in another PR

Reply:
I do it in mine ;)